packages/babel-node/test/index.js

```diff
@@ -15,20 +16,10 @@ const fileFilter = function(x) {
 };
 
 const outputFileSync = function(filePath, data) {
-  fs.mkdirSync(path.dirname(filePath), { recursive: true });
+  makeDirSync(path.dirname(filePath));
```

Comment:
Curious, why this switch?

Reply:
@existentialism This must have come from my attempt to pull changes from `master`. Let me fix it.

Reply:
Fixed.